generators/app/templates/client/package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "<%= _.kebabCase(appName) %>",
   "dependencies": {
+    "react-hot-loader": "^3.0.0-beta.3",
+    "style-loader": "^0.13.1",
+    "babel-core": "^6.0.20",
+    "babel-eslint": "^4.1.3",
     "babel-loader": "^6.2.4",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-polyfill": "^6.9.1",
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.11.1",
-    "babel-preset-stage-1": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
```

Comment:
I don't really like the idea of using the `stage-0` preset. It is unstable, it may contain plugins whose behavior will change, it may contain plugins that will transform syntax that ends up not even being Javascript at all.

If that happens, the project may end up being written in some frankenscript that is not old Javascript and is not new Javascript and must use a specific (probably unsupported by then) babel plugin to work.

I think the ideal stage preset in terms of safety is the `stage-2` preset, but I think `stage-1` is a good compromise, as the chances of a proposal changing in an extreme way once they reach stage 1 are very low, and even lowest are the chances of it being scrapped altogether.